packages/control-property-editor/src/panels/outline/Tree.tsx

```diff
@@ -384,6 +390,7 @@ export const Tree = (): ReactElement => {
         );
 
         const tooltipId = `tooltip--${groupName}`;
+        const headerName = hasDefaultContent ? `${groupName} (has default content)` : groupName;
 
```

Comment:
Shouldn't we use texts from i18n?

Reply:
Added in: [c86f608](https://github.com/SAP/open-ux-tools/pull/1867/commits/c86f608325dd2d9f9b520beffc14d1a6122242c5)